src/Billing/Billing/Az.Billing.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.2'
+ModuleVersion = '2.0.0'
```

Comment:
Azure powershell team will update the version in release process

Reply:
Sure, Done.
Please upgrade the major version since we are deprecating old cmdlets which are not supported anymore.